### ISSUE TITLE
feature request fully local

### ISSUE THREAD

**Issue by BlackKite84** [shinsenter/defer.php#30]
i wish me a fully local version of JS because General Data Protection Regulation GDPR (DSGVO)
i cant use external source and the Polyfill.js is includes as External source by me

**Comment by shinsenter** [comment]
@BlackKite84 You can download both defer_plus.min.js and polyfill.min.js and put these files into your local project.
```
https://cdn.jsdelivr.net/npm/@shinsenter/defer.js@1.1.6/dist/defer_plus.min.js
https://cdn.jsdelivr.net/npm/@shinsenter/defer.js@1.1.6/dist/polyfill.min.js
```

**Comment by BlackKite84** [comment]
This is my Folder
![002](https://user-images.githubusercontent.com/3486554/56514794-8bd2d400-6536-11e9-93ed-045d6681cf11.PNG)

This ist the SourceCode
![001](https://user-images.githubusercontent.com/3486554/56514792-8bd2d400-6536-11e9-9e57-95303c26e8d7.PNG)


Sittings

> 			// Library injection
>			$defer->append_defer_js     = true;

**Comment by shinsenter** [comment]
I will add another option not to insert defer.js, but you must include the script tags yourself. It will be released in the next version.

**Comment by shinsenter** [comment]
@BlackKite84 
The option was added in version 1.0.11:
https://github.com/shinsenter/defer.php/tree/1.0.11

```php
$defer->manually_add_deferjs   = true;
```

**Comment by BlackKite84** [comment]
i thinks this line
https://github.com/shinsenter/defer.php/blob/a708237b01810771ad8e989d5dfe960ea95ae388/src/DeferOptimizer.php#L71

remove the Js code when i include this in my document 

<script id="defer-js" src="...defer_plus.min.js"></script>
<script id="polyfill-js" src="....polyfill.min.js"></script>

**Comment by BlackKite84** [comment]
and with the option "fix_render_blocking" the script ist Moved to footer

**Comment by shinsenter** [comment]
You are right, bro.
Thank you for your feedback and I will improve this problem asap.

**Comment by shinsenter** [comment]
@BlackKite84 
The issue was fixed in #39
https://github.com/shinsenter/defer.php/releases/tag/1.0.12
